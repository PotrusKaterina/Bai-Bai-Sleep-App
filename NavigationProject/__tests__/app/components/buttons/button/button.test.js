import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Button from "../../../../../app/components/buttons/button/button";


describe('AuthInput', () => {
    test('create snapshot AuthInput', () => {
        const render = new ShallowRenderer();
        const snapshot = render.render(<Button />);
        expect(snapshot).toMatchSnapshot();
    })
});