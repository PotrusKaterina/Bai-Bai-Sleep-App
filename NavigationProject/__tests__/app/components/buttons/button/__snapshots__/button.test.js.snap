// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthInput create snapshot AuthInput 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#424242",
      "borderRadius": 4,
      "margin": 5,
      "padding": 10,
      "width": 160,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontWeight": "bold",
      }
    }
  />
</TouchableOpacity>
`;
